### ISSUE TITLE
module declares its path as: github.com/Shopify/sarama but was required as: github.com/IBM/sarama

### ISSUE THREAD

**Issue by geberl** [IBM/sarama#2491]
Currently it's not possible to import the migrated module.

```
go mod tidy
(...)
go: git.my.company/myproject/internal/cmd imports
        github.com/IBM/sarama: github.com/IBM/sarama@v1.38.1: parsing go.mod:
        module declares its path as: github.com/Shopify/sarama
                but was required as: github.com/IBM/sarama
```

I guess line 1 in go.mod needs to be adjusted, and a new version needs to be published as well.

```
module github.com/Shopify/sarama
```

**Comment by dnwe** [comment]
Yes the migration is still work in progress at the moment. You should still be able to pull the module from the gocache at the old import path?

**Comment by edoger** [comment]
> Yes the migration is still work in progress at the moment. You should still be able to pull the module from the gocache at the old import path?

Can module names be lowercased?  github.com/ibm/sarama   ?

**Comment by softwarespot** [comment]
Will https://pkg.go.dev/github.com/IBM/sarama also be updated too? As it currently references an old version?

**Comment by dnwe** [comment]
Yes that’s just because pkg.go.dev auto imported all version tags until a go.mod was added I think

**Comment by softwarespot** [comment]
Thanks @dnwe and a big thanks to you and IBM for continuing to support this project. I no doubt will start to contribute once things have been finalized your end.

**Comment by dnwe** [comment]
Closing as resolved by v1.40.0 release
